### PR TITLE
feat: don't stop chasing waterfalls

### PR DESCRIPTION
**Pull request by pauldambra**
## Problem

The change to the web performance schema meant the waterfall page stopped working

## Changes

* adds a query/API end point
* de-ANTs and updates the waterfall page
* data-exploration-ifies the waterfall page

waterfalls

**NB this was originally a stacked PR, there aren't really 44 commits here**

## How did you test this code?

running locally

### CODE REVIEW

File: ee/api/performance_events.py [hunk 1]
```diff
@@ -157,3 +207,15 @@ def list(self, request: request.Request, *args, **kwargs) -> Response:
...
 
         return Response({"results": serializer.data})
+
+    @action(methods=["GET"], detail=False)
+    def recent_pageviews(self, request: request.Request, *args, **kwargs) -> Response:
+        params_serializer = RecentPageViewPerformanceEventsQuerySerializer(data=request.GET)
+        params_serializer.is_valid(raise_exception=True)
+        params = params_serializer.validated_data
+
+        results = RecentPageViewPerformanceEvents.query(self.team_id, number_of_days=params.get("number_of_days"))
```
Comment (benjackwhite) [inline ee/api/performance_events.py:217]: Totally up for this given it is all just being used by us atm but before we go live with any of this we should probably try and conform to the typical API `date_from`

File: frontend/src/scenes/performance/WebPerformance.tsx [hunk 2]
```diff
@@ -29,11 +31,61 @@ export const webPerformancePropertyFilters: AnyPropertyFilter[] = [
...
+    )
+}
+
 const EventsWithPerformanceTable = (): JSX.Element => {
-    const { setEventToDisplay } = useActions(webPerformanceLogic)
+    const { recentPageViews, recentPageViewsLoading } = useValues(webPerformanceLogic)
     const { featureFlags } = useValues(featureFlagLogic)
     const featureDataExploration = featureFlags[FEATURE_FLAGS.DATA_EXPLORATION_LIVE_EVENTS]
 
+    const oldFashionedColumns: LemonTableColumn<
```
Comment (benjackwhite) [inline frontend/src/scenes/performance/WebPerformance.tsx:55]: _tips hat_

File: frontend/src/styles/utilities.scss [hunk 3]
```diff
@@ -313,35 +313,46 @@
...
-    }
-    .#{$char}-2\/5 {
-        #{$kind}: 40%;
-    }
-    .#{$char}-3\/5 {
-        #{$kind}: 60%;
-    }
-    .#{$char}-4\/5 {
-        #{$kind}: 80%;
+    @each $variant in ('', 'max-', 'min-') {
```
Comment (benjackwhite) [inline frontend/src/styles/utilities.scss:316]: oooh clever

**Review by benjackwhite — APPROVED** [review]
Looks good. I didn't go in too deep with the review and focused on mostly making sure it overall works so we can get it back up and running for testing. 

I do think we may want to revisit some bits (in particular the API) before it lands with too many "real" users 😉

File: ee/api/performance_events.py (hunk 1 above)
Comment (pauldambra) [reply ee/api/performance_events.py:217]: fair! 👍

**Pull request merged**